css/src/new-redirects.css

```diff
@@ -61,6 +61,22 @@ body.seo_page_wpseo_redirects {
 	}
 
 	.yst-root {
+		.yst-notifications {
+			max-height: calc(100% - 4rem - 32px); /* Extra 32px for the height of the WP admin bar. */
+
+			@media (max-width: 782px) {
+				max-height: calc(100% - 4rem - 48px); /* Extra 48px for the height of the mobile WP admin bar. */
+			}
+		}
+
+		.yst-notifications--bottom-left {
+			z-index: 9991; /* 1 above admin menu */
```

Comment:
Maybe you can use even lower value like 10, needs to be tested. https://tailwindcss.com/docs/z-index#examples
```suggestion
			@apply yst-z-50; /* 1 above admin menu */
```